shared/styles/index.native.tsx

```diff
@@ -66,7 +66,7 @@ export {isDarkMode} from './dark-mode'
 export const collapseStyles = (
   styles: ReadonlyArray<CollapsibleStyle>
 ): ReadonlyArray<Object | null | false | void> => {
-  return styles
+  return styles.filter(Boolean)
```

Comment:
just kills empty values, which are benign to RN but makes the shots less and maybe tinier object to send over the bridge